### PR TITLE
adjust styling and responsiveness

### PR DESCRIPTION
**Pull request by Henning456**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,9 +1,9 @@
+:root {
+  background-color: #0a741f;
+}
+
 .App {
   text-align: center;
-  /* background-color: rgb(187, 106, 8); */
```
Comment (NatalieLuka) [inline src/App.css:3]: Will you use this later ?

File: src/App.css [hunk 2]
```diff
@@ -61,11 +62,3 @@ button {
   top: 0;
   left: 0;
 }
-
```
Comment (NatalieLuka) [inline src/App.css:64]: animation should be done later Henning ! 🤌🏼

**Review by NatalieLuka — APPROVED** [review]
Incredible styling !

**Review by Staicy-S — APPROVED** [review]
Nice job Henning! As long as it looks good, I agree with your changes :) And if you want to share some of your jsx code as a PR with us, we would love to take a look as well 😊

**Pull request merged**